backend/tracim_backend/config.py

```diff
@@ -1400,6 +1400,10 @@ def configure_filedepot(self) -> None:
                 "depot.backend": DepotFileStorageType.MEMORY.depot_storage_backend
             }
 
+        # INFO - G.M - 2022-04-08 - Clear Depot Manager to avoid issue with
+        # Tracimcli interactive shell.
+        DepotManager._clear()
+
```

Comment:
```suggestion
        # INFO - G.M - 2022-04-08 - Clear Depot Manager to avoid issue with
        # Tracimcli interactive shell.
        DepotManager._clear()
```

I see no reason this should break something, but verification with CI is required here.